src/main/java/ezschedule/logic/commands/AddCommand.java

```diff
@@ -46,8 +49,11 @@ public CommandResult execute(Model model) throws CommandException {
         if (model.hasEvent(toAdd)) {
             throw new CommandException(MESSAGE_DUPLICATE_EVENT);
         }
-
+        model.recentEvent().clear();
+        model.recentCommand().clear();
         model.addEvent(toAdd);
+        model.recentEvent().add(toAdd);
+        model.recentCommand().add(this);
```

Comment:
This part is generally fine. Perhaps can include a method in model for better abstraction? Instead of 
```
model.recentEvent().clear();
model.recentCommand().clear();
```
perhaps try
```
model.clearRecent();
```